nilearn/__init__.py

```diff
@@ -45,7 +45,7 @@ def _py2_deprecation_warning():
     warnings.simplefilter('once')
     py2_warning = ('Python2 support is deprecated and will be removed in '
                    'a future release. Consider switching to Python3.')
-    if six.PY2:
+    if LooseVersion(sys.version) < LooseVersion('3'):
```

Comment:
The best way to do this is to use sys.version_info. It gives safer and simpler comparisons.

Reply:
Cool! I don't even need LooseVersion. Thanks.